src/styles/images/index.md.njk

```diff
@@ -8,53 +8,214 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-Avoid unnecessary decoration. Only use images if there’s a real user need.
+Avoid using images for unnecessary decoration. Only use images if there’s a real user need.
 
-## Aspect ratio
+Make sure any information contained in an image can be understood by someone who cannot see it. Also consider partially sighted users with visual impairments and the assistive technologies they might use.
 
-### Photography
+This guidance is for government teams that build online services. To learn how to use images as a GOV.UK content publisher, go to [GOV.UK content guidance for images](https://www.gov.uk/guidance/content-design/images).
 
-The aspect ratio for photographs should be 3:2.
+## Using images to give information
 
-{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+Services usually work best without relying on images. Focus on writing clear, simple content for your service first.
 
+Adding images can sometimes help users better understand instructions. However, images should only be visual aids — written content must contain all the information the user needs to complete the service.
 
-### Illustrations or representative imagery
+If you choose to add an image to your service, you’ll need to carefully decide if the image needs to be described with alternative text (alt text).
 
-If your image represents something physical, such as a letter, document or credit card you should use the aspect ratio of that item.
+## Photography
 
-{{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
+Use photography when it’s important to show a lifelike representation of something. For example, showing a preview of a document a user has uploaded.
 
+Do not use photography to represent abstract concepts, such as to convey an impression or emotion to the user. Avoid using generic stock photography in your service.
 
-## Alternative text
+## Illustrations
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+Use illustrations to show a simplified image of something complex. For example:
+- to demonstrate a physical interaction, like scanning a passport
+- to help users find something on a document, like a reference number
 
-Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
+### Creating illustrations
 
-All images, except decorative images, must have alt text that:
+Use as few visual elements as possible to communicate your message.
+
+Start by following the guidance on [Colour and using the GOV.UK colour palette](/styles/colour/).
```

Comment:
should this be a lowercase colour not Colour?

Reply:
I'll change this to lowercase — when we're referring to a page title, it's capitalised... but as written it's read both ways. I think lowercase looks better in this instance too.